stack.yaml

```diff
@@ -15,7 +15,7 @@
 # resolver:
 #  name: custom-snapshot
 #  location: "./custom-snapshot.yaml"
-resolver: lts-14.9
+resolver: lts-15.3
```

Comment:
Is this because of the optparse lib? I get a warning from stack:

```
Stack has not been tested with GHC versions above 8.6, and using 8.8.2, this may fail
```

Reply:
Same here, we both have an old `stack` executable most probably.

Reply:
I bumped because `optparse` introduced a dependency that failed with old `ansi-terminal` IIRC.

Reply:
OK, I'll try to upgrade my Stack.

Reply:
That worked, so I think we're good.